sdk/kiosk/src/utils.ts

```diff
@@ -199,6 +202,33 @@ export async function getAllDynamicFields(
 	return data;
 }
 
+/**
+ * A helper to fetch all objects that works with pagination.
+ * It will fetch all objects in the array, and limit it to 50/request.
+ * Requests are sent using `Promise.all`.
+ */
+export async function getAllObjects(
+	client: SuiClient,
+	ids: string[],
+	options: SuiObjectDataOptions,
+	limit: number = DEFAULT_QUERY_LIMIT,
+) {
+	let chunks = Array.from({ length: Math.ceil(ids.length / limit) }, (_, index) =>
```

Comment:
You should prefer const over let everywhere here. We need to set up a lint rule for this.